third_party/silabs/silabs_board.gni

```diff
@@ -48,6 +48,9 @@ declare_args() {
 
   # Disable UART log forwarding by default
   sl_uart_log_output = false
+
+  # Self-provision enabled
+  use_provision_channel = false
```

Comment:
That would work, but then we'll need to add it to SiWx917_sdk.gni too. Since this argument is common to all boards, I think this is the right place to declare it.